src/common-components/RedirectLogistration.jsx

```diff
@@ -10,7 +10,13 @@ function RedirectLogistration(props) {
     // If we're in a third party auth pipeline, we must complete the pipeline
     // once user has successfully logged in. Otherwise, redirect to the specified redirect url.
     if (finishAuthUrl) {
```

Comment:
I guess we could combine the second condition with a not (!) operator here:
```suggestion
    if (finishAuthUrl && !redirectUrl.includes(finishAuthUrl)) {
```